actors/market/Cargo.toml

```diff
@@ -15,7 +15,7 @@ crate-type = ["cdylib", "lib"]
 
 [dependencies]
 fil_actors_runtime = { version = "10.0.0-alpha.1", path = "../../runtime"}
-
+fil_actor_miner = { path = "../miner" }
```

Comment:
Please don't add this dependency, but duplicate the necessary param/return structures into `market/ext.rs`, replacing those already there for GetControlAddresses.

Reply:
removed